static/js/src/canonical-cla/utils/api.ts

```diff
@@ -100,7 +100,7 @@ const request = async <M extends HttpMethod, E extends EndpointForMethod<M>>(
     requestUrl = `${endpoint}?${params.toString()}`;
   }
   const encodedRequestUrl = btoa(requestUrl);
-  const url = `/legal/contributors/agreement/api?request_url=${encodedRequestUrl}`;
+  const url = `https://canonical.com/legal/contributors/agreement/api?request_url=${encodedRequestUrl}`;
```

Comment:
canonical-cla folder should be removed from this project along with canonical-cla references in webapp for python code.